pkg/apis/v1/nodepool_conversion_test.go

```diff
@@ -269,6 +269,24 @@ var _ = Describe("Convert V1 to V1beta1 NodePool API", func() {
 			Expect(v1beta1nodepool.Status.Resources[resource]).To(Equal(v1nodepool.Status.Resources[resource]))
 		}
 	})
+	Context("Round Trip", func() {
```

Comment:
Can we do some testing here with the `nil (Never)` value? This should be more important for ConsolidateAfter, since it's going from a pointer to a non-pointer.

Reply:
I don't think it makes a difference for ConsolidateAfter since the wrapped duration is a pointer regardless, but I can go ahead and add a test. I don't think it's something we explicitly need to test since it only has one valid representation ("Never") but it doesn't hurt to add.